theme.js

```diff
@@ -2327,6 +2327,10 @@ export default () => {
         height: responsiveWidth(194),
         paddingBottom: resolveVariable('smallGutter'),
       },
+      // Just enough to display portrait mode Vimeo videos and all Vimeo player controls
+      vimeoVideo: {
+        height: responsiveWidth(220),
```

Comment:
```suggestion
        height: responsiveHeight(220),
```
Are we sure we want to use `responsiveWidth` (note **_width_**) for `height`?

Added replacement with `responsiveHeight` instead, commit it if necessary, if not, ignore this comment.